pkg/scheduling/pd/scheduler_test.go

```diff
@@ -213,10 +191,8 @@ func TestPDSchedule(t *testing.T) {
 				t.Errorf("Unexpected error, got %v", err)
 			}
 
-			datastore := &fakeDataStore{pods: test.input}
 			scheduler := scheduling.NewSchedulerWithConfig(schedulderConfig)
```

Comment:
The configuration needs to be loaded from text, as of PR #214

Reply:
I don't agree with this statement - 
this unit test is meant to test the scheduler unit while completely decoupling from how the scheduler config was initialized. will handle this in a follow up.